teams/vsp/teams/tools/backend/external-service-monitoring/bin/capture-slo-screenshots

```diff
@@ -2,39 +2,40 @@
 
 # frozen_string_literal: true
 
-SCRIPT_VERSION = '0.0.2'
-PROFILE_NAME   = ENV['PROFILE_NAME'] || 'dev-edition-default'
-EXTENSION_NAME = '/extensions/switchyomega@feliscatus.addons.mozilla.org.xpi'
-GRAFANA_BASE_URL = 'http://grafana.vfs.va.gov/d/pEgVdRlZk/external-service-performance-indicators'
+SCRIPT_VERSION     = '0.0.3'
+PROFILE_NAME       = ENV['PROFILE_NAME'] || 'dev-edition-default'
+GRAFANA_BASE_URL   = 'http://grafana.vfs.va.gov/d/pEgVdRlZk/external-service-performance-indicators'
 LINUX_FIREFOX_PATH = '/opt/firefox/firefox'
-OSX_FIREFOX_PATH = '/Applications/Firefox.app/Contents/MacOS/firefox'
+OSX_FIREFOX_PATH   = '/Applications/Firefox.app/Contents/MacOS/firefox'
 
 require 'bundler/inline'
 require 'rbconfig'
 gemfile do
   source 'https://rubygems.org'
 
-  ruby RUBY_VERSION || '2.6.6'
+  ruby RUBY_VERSION || '2.6'
 
   gem 'json'
   gem 'pastel'
   gem 'pry'
   gem 'tty-spinner'
   gem 'watir'
-  gem 'webdrivers', '4.0.0'
+  gem 'webdrivers', '4.4.1'
 end
 
 # SLOReporter
 class SLOReporter
-  attr_accessor :browser, :backend, :board_url
+  attr_accessor :browser, :backend, :board_url, :tmpfile
+
   def initialize(browser, backend)
-    @browser  = browser
-    @backend  = backend
+    @browser, @backend = browser, backend
     @board_url = GRAFANA_BASE_URL + query_params
+    @tmpfile   = 'tmp/screenshots/' + backend + 'tmp.png'
   end
 
   def query_params
-    "?orgId=1&var-backend=#{backend}"
+    base = "?orgId=1&var-backend=#{backend}"
```

Comment:
no need for an assignment here.  looks like this might be leftover from some debugging?